src/gromacs/nblib/molecules.h

```diff
@@ -83,7 +83,10 @@ class Molecule
 public:
     Molecule(std::string moleculeName);
 
-    // Add a particle to the molecule with full specification of parameters.
+    //! The molecule name
+    std::string name() const;
```

Comment:
A bit trivial but it would be slightly nicer to have this later in the class definition, with the other getters.

Reply:
sure